nova/virt/vmwareapi/images.py

```diff
@@ -374,7 +375,7 @@ def fetch_image_stream_optimized(context, instance, session, vm_name,
 
 
 def _import_image(session, read_handle, vm_import_spec, vm_name, vm_folder_ref,
-                  res_pool_ref, file_size):
+                  res_pool_ref, datastore_ref, file_size):
```

Comment:
This new parameter is unused.

Reply:
You are right. Thanks. It looks, I didn't clean the code up properly. I've done that now and the change is much smaller.